From 85010a250c9ba7688c4028d678f3b45ccef6f8e7 Mon Sep 17 00:00:00 2001
From: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Date: Mon, 1 Apr 2019 10:50:07 -0700
Subject: [PATCH 0590/1852] ASoC: xlnx: change HDMI audio mclk

commit f3efb86108365cec7bc00293440df0878d537f2f from
https://github.com/Xilinx/linux-xlnx.git

audio mclk depends on the ACR IP being used. Multiplier value is changed
in updated version of ACR IP. Updated the same in driver.

Signed-off-by: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 sound/soc/xilinx/xlnx_pl_snd_card.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/xilinx/xlnx_pl_snd_card.c b/sound/soc/xilinx/xlnx_pl_snd_card.c
index ab54827a677c..ab422771c53f 100644
--- a/sound/soc/xilinx/xlnx_pl_snd_card.c
+++ b/sound/soc/xilinx/xlnx_pl_snd_card.c
@@ -90,7 +90,7 @@ static int xlnx_hdmi_card_hw_params(struct snd_pcm_substream *substream,
 	case 96000:
 	case 176400:
 	case 192000:
-		prv->mclk_ratio = 512;
+		prv->mclk_ratio = 768;
 		break;
 	default:
 		return -EINVAL;
-- 
2.31.1

